pyproject.toml

```diff
@@ -54,3 +54,23 @@ changelog = "https://github.com/jazzband/django-constance/releases/"
 
 [tool.setuptools.packages.find]
 include = ["constance*"]
+
+
+[tool.ruff]
+# Same as Black.
+line-length = 88
+indent-width = 4
+
+[tool.ruff.format]
+quote-style = "double"
```

Comment:
project is mostly written with single quotes, let`s leave status quo for this.

Reply:
Most Python project I know use double quote and tabs, doesn't it?

Reply:
It doesn't matter to me https://github.com/jazzband/django-constance/pull/560/commits/a40c0ab7c2f14fe19744a59d9ec0f81a85596fce
changed

Reply:
can revert again if there is sense 😄